experimental/PropertyDDS/examples/partial_checkout/.npmignore

```diff
@@ -0,0 +1,6 @@
+nyc
```

Comment:
I think there are few packages which still miss `.npmignore`

Reply:
I added it to most since there was a warning about the file missing. changeset, common and properties did not have this warning, but I assume that is because those don't use an tsc build step and thus won't have temporary files lying in the folder.